changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,39 @@ Cisco Iosxr Collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
+-------------
+
```

Comment:
@ashwini-mhatre @justjais The major change needs to have caching related feature and direct execution as part of the changelog.